lms/static/sass/discussion/views/_thread.scss

```diff
@@ -106,24 +72,124 @@ body.discussion, .discussion-module {
     }
 
     .comment-actions-list {
+      @include float(right);
       display: inline-block;
       width: flex-grid(2,12);
       vertical-align: top;
     }
 
-    //TO-DO : clean up posted-details styling, currently reused by responses and comments
     .posted-details {
       margin-top: 0;
     }
   }
 }
 
+// +thread - wrapper styling
 .forum-thread-main-wrapper {
+  @include border-radius(3px, 3px, 0, 0);
   border-bottom: 1px solid $white; // Prevent collapsing margins
-  border-radius: 3px 3px 0 0;
   background-color: $white;
 }
 
+// +thread - elements - shared styles
+body.discussion {
+
+  .discussion-post, .discussion-response, .discussion-comment {
+    
+    // thread - images
+    .author-image {
+      @include margin-right($baseline/2);
+      display: inline-block;
+      background-color: $gray-l4;
```

Comment:
Is this needed (assuming the image will scale to take up the entirety of the dimensions)?